auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div id="continuation-application-review-table">
-    <v-form class="continuation-application-search continuation-application-search">
+    <v-form class="fas-search continuation-review-search">
```

Comment:
Re: "fas-seach", the name doesn't make sense. It's imported from '~/fas-ui/src/assets/scss/search.scss';
But if I extend it or rename it, the style is lost. I keep it here so the search column style is correct.

Reply:
This is a dependency on an external project that we really shouldn't have, but I realize the staff account table components use this, too.

I suggest we wait until we port this Entity component to a different project to resolve this CSS issue.